### DISCUSSION TITLE
Could BookWyrm have a simple logo?

### DISCUSSION THREAD

**Discussion by placoderm** [bookwyrm-social/bookwyrm#2430] [Ideas]
I'd love to add my BW account to my website in the social icons section, e.g.

![icons](https://user-images.githubusercontent.com/31570884/203033641-f6834389-52fa-4635-bc62-79e6e227f591.png)

But the current logo for BW is kind of complicated. Have you considered something simpler that could match other social icons?

**Comment by mouse-reeve** [comment]
It would be wonderful to have a logo or a version of the logo that would read as an icon. There is a 0% chance of _me_ making it, but I'd love to have it :)

**Reply by placoderm** [reply]
Would it be ok if I posted about it on Mastodon? Seems like there must be someone out in the fediverse that could create one. Of course it would be up to you to accept one that you liked, or accept none at all.

Btw, is does the dragon in your current image have a name? I'm assuming he is a kind of a mascot.

Thanks again for all your hard work.

**Reply by placoderm** [reply]
OK, my graphic design skills are crap, but looking at book and worm icons already out there, I threw this together. Obviously someone with skill would have to redo it/improve it.

![image](https://user-images.githubusercontent.com/31570884/203528258-5cc9cca7-3b52-4cef-b3ef-619b276bc492.png)

The worm makes a W. The worm has also chewed a self-size hole through the book as both human and actual book worms tend to do. And the worm kind of looks like a bookmark too.

@mouse-reeve, I know that you have way more important things to worry about. So feel free to ignore.

**Reply by jaschaurbach** [reply]
Oh I like this idea - unfortunately I have no graphic skills as well :D

**Comment by andypiper** [comment]
Just looking around for an SVG icon of the Bookwyrm logo, and came to this discussion. I like the ideas shared by @placoderm here. Would be great to have something shared by the project for general usage 🙂

**Comment by roke-julian-lockhart** [comment]
> https://github.com/bookwyrm-social/bookwyrm/discussions/2430#discussion-4586202

@placoderm, perhaps this should be converted into an issue.

**Comment by sciencefictionspacecraft** [comment]
I'm a graphic designer, and I love placoderm's idea. 

I could work it up...

**Reply by roke-julian-lockhart** [reply]
> https://github.com/bookwyrm-social/bookwyrm/discussions/2430#discussioncomment-6499705

@Retr0mantic, please do.

**Comment by em3link** [comment]
I think it is not perfect (probably not that great as a very small icon), but here is my take on a Bookwyrm icon.
Even though the suggestion by @placoderm is quite interesting and could be explored, I must say I prefer dragons.
Hence my take on a Bookwyrm icon - I tried to stay close to the existing logo and simplify it a bit.

Here is what it looks like:
![bookwyrm_icon_demo](https://github.com/bookwyrm-social/bookwyrm/assets/45605175/610563cc-56e8-4ef8-8d3c-3b169b6754f3)

Feel free to use it, share it, modify it as you like.

Here is the (optimized) svg version:
![bookwyrm](https://github.com/bookwyrm-social/bookwyrm/assets/45605175/d6039f4b-ddc9-43e6-b0b2-bc41303df276)

**Reply by roke-julian-lockhart** [reply]
> https://github.com/bookwyrm-social/bookwyrm/discussions/2430#discussioncomment-8699302

@em3link, I like it a lot.

**Reply by mouse-reeve** [reply]
I just realized I only emoji responded to this, but it warrants a proper comment -- I think this is absolutely perfect, thank you so much!

**Reply by andypiper** [reply]
maybe worth updating the logo used on the GitHub org page for bookwyrm-social? And also maybe updating it so the org page also has a link to Bookwyrm's canonical Mastodon account, now that GH allows more social links to be added.

**Reply by pepev-nrt** [reply]
I think that logo is just perfect, this is how it looks in the social icons of my webpage. Is it ok to submit it to [SVG Repo](https://www.svgrepo.com/) to be more accessible to everybody?
![image](https://github.com/user-attachments/assets/83649852-0231-4acd-bd22-93e01e36ca59)

**Reply by mouse-reeve** [reply]
@pepev-nrt please do

**Reply by em3link** [reply]
@pepev-nrt No problem on my side either, this is a good idea. 😊
